db.json

```diff
@@ -1,68 +1,344 @@
+        }
+      ],
+      "total": "10000",
+      "status": "NEW",
+      "userId": "U5312"
+    },
+    {
+      "id": "S1005",
+      "name": "New Furnitures From Chittagong",
+      "cargo": [
+        {
+          "type": "Couch22 White",
+          "description": "103 Couch model Couch22",
+          "volume": "103"
+        },
+        {
+          "type": "Couch22 Black",
+          "description": "103 Couch model Couch22",
+          "volume": "103"
+        }
+      ],
+      "mode": "air",
+      "type": "FCL",
+      "destination": "Mullerstr 6, 13353 Berlin",
+      "origin": "Chittagong Port",
+      "services": [
+        {
+          "type": "insurance"
+        }
+      ],
+      "total": "1000",
+      "status": "NEW",
+      "userId": "U5312"
+    },
+    {
+      "id": "S1006",
+      "name": "Vacuum Cleaners",
+      "cargo": [
+        {
+          "type": "TVAC515",
+          "description": "555 TVAC515 120V",
+          "volume": "555"
+        },
+        {
+          "type": "Couch22 Black",
+          "description": "555 TVAC515 240V",
+          "volume": "555"
+        }
+      ],
+      "mode": "air",
+      "type": "FCL",
+      "destination": "Tughlakabad Port",
+      "origin": "Hamburg Port",
+      "services": [
+        {
+          "type": "insurance"
+        },
+        {
+          "type": "customs"
+        }
+      ],
+      "total": "100000",
+      "status": "ACTIVE",
+      "userId": "U531221"
+    },
+    {
+      "id": "S1007",
+      "name": "Mattress",
+      "cargo": [
+        {
+          "type": "MAT128",
+          "description": "MAT12ftx82ft",
+          "volume": "666"
+        }
+      ],
+      "mode": "air",
+      "type": "FCL",
+      "destination": "Shengen Port",
+      "origin": "Bremerhaven Port",
+      "services": [
+        {
+          "type": "customs"
+        }
+      ],
+      "total": "101010",
+      "status": "ACTIVE",
+      "userId": "U531222"
+    },
+    {
+      "id": "S1008",
+      "name": "Scooters",
+      "cargo": [
+        {
+          "type": "Gororo",
+          "description": "Gororo Type B",
+          "volume": "150"
+        }
+      ],
+      "mode": "air",
+      "type": "FCL",
+      "destination": "Rotterdam Port",
+      "origin": "Xianbou Port",
+      "services": [
+        {
+          "type": "customs"
+        }
+      ],
+      "total": "60120120",
+      "status": "COMPLETED",
+      "userId": "U1337"
+    },
+    {
+      "id": "S1009",
+      "name": "Helmets",
+      "cargo": [
+        {
+          "type": "HelmGoro",
+          "description": "HelmGoro Red",
+          "volume": "150"
+        }
+      ],
+      "mode": "air",
+      "type": "LCL",
+      "destination": "Rotterdam Port",
+      "origin": "Xianbou Port",
+      "services": [
+        {
+          "type": "customs"
+        }
+      ],
+      "total": "60120120",
+      "status": "COMPLETED",
+      "userId": "U1337"
+    },
+    {
+      "id": "S10110",
```

Comment:
Let's try to make the shipment ids sequential so it's easy to count. Maybe provide 30 shipments